src/containers/account/AccountContainer.js

```diff
@@ -52,14 +79,6 @@ const AccountContainer = () => {
     dispatch(clearAtlasCredentials());
   }, [dispatch]);
 
-  if (getAtlasCredentialsRS === RequestStates.PENDING || getAtlasCredentialsRS === RequestStates.STANDBY) {
-    return (
-      <AppContentWrapper>
-        <Spinner />
-      </AppContentWrapper>
-    );
-  }
```

Comment:
Why did you decide to remove this?

Reply:
I decided to remove it because when you regenerate the atlas credential, you have to call `getAtlasCredentials`, which makes the whole page really jerky with that spinner. Decided to use these loading states to disable the buttons instead. I like your idea above though. I'll change the prop to `isLoading`.